docs/gh-action.rst

```diff
@@ -0,0 +1,136 @@
+==============================
+Using Bowtie in GitHub Actions
+==============================
+
+You can now use bowtie as a GitHub Action. Follow the steps to get started with it in a few simple steps:
+
+
+Setup Bowtie
+------------
+
+1. Initial Steps
+^^^^^^^^^^^^^^^^
+
+* Create ``.github/workflows`` directory in your repository.
+* Create a file with ``.yml`` extension.
+* Enter the following code in the YAML file:
+
+.. code:: yaml
+
+    name: Setup Bowtie
+    run-name: ${{ github.actor }} is learning to use Bowtie
+
+    on: [push]
+
+    jobs:
+    sets-up-bowtie:
+        runs-on: ubuntu-latest
+
+        steps:
+        - name: Install Bowtie
+            uses: bowtie-json-schema/bowtie@v2023.05.12
+
+
+*The code snippet will install bowtie every time you push onto the repository.*
+
+2. Verification Steps
```

Comment:
I think it's OK for people to skip this step in general and not print the version, that's perhaps mostly useful just during debugging, but otherwise there's no reason to always run it, the version will match what you use in the `uses` line, i.e. `bowtie-json-schema/bowtie@v2023.05.12` should generally give you a `--version` of `v2023.05.12``.

So think you can likely skip this section.